### ISSUE TITLE
[Bug]: New install of v0.3.1 requires v0.3.0 first, default config parser error

### ISSUE THREAD

**Issue by hasso** [tubearchivist/tubearchivist#408]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

Synology

## Describe the bug

In fresh install on Synology, tubearchivist container doesn't start. 

### Relevant log output

```shell
2023-01-15T09:58:37.960426954Z  stdout  {
2023-01-15T09:58:37.961005062Z  stdout    "name" : "tubearchivist-es",
2023-01-15T09:58:37.961083725Z  stdout    "cluster_name" : "docker-cluster",
2023-01-15T09:58:37.961115819Z  stdout    "cluster_uuid" : "<stripped>",
2023-01-15T09:58:37.961147811Z  stdout    "version" : {
2023-01-15T09:58:37.961177070Z  stdout      "number" : "8.6.0",
2023-01-15T09:58:37.961205833Z  stdout      "build_flavor" : "default",
2023-01-15T09:58:37.961239816Z  stdout      "build_type" : "docker",
2023-01-15T09:58:37.961266035Z  stdout      "build_hash" : "f67ef2df40237445caa70e2fef79471cc608d70d",
2023-01-15T09:58:37.961301962Z  stdout      "build_date" : "2023-01-04T09:35:21.782467981Z",
2023-01-15T09:58:37.961332193Z  stdout      "build_snapshot" : false,
2023-01-15T09:58:37.961363269Z  stdout      "lucene_version" : "9.4.2",
2023-01-15T09:58:37.961391910Z  stdout      "minimum_wire_compatibility_version" : "7.17.0",
2023-01-15T09:58:37.961423165Z  stdout      "minimum_index_compatibility_version" : "7.0.0"
2023-01-15T09:58:37.961449914Z  stdout    },
2023-01-15T09:58:37.961474563Z  stdout    "tagline" : "You Know, for Search"
2023-01-15T09:58:37.961503594Z  stdout  }
2023-01-15T09:58:42.169930994Z  stdout  Traceback (most recent call last):
2023-01-15T09:58:42.170307879Z  stdout    File "/app/manage.py", line 23, in <module>
2023-01-15T09:58:42.170357140Z  stdout      main()
2023-01-15T09:58:42.170385067Z  stdout    File "/app/manage.py", line 19, in main
2023-01-15T09:58:42.170422601Z  stdout      execute_from_command_line(sys.argv)
2023-01-15T09:58:42.170453222Z  stdout    File "/root/.local/lib/python3.10/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
2023-01-15T09:58:42.171837721Z  stdout      utility.execute()
2023-01-15T09:58:42.171885637Z  stdout    File "/root/.local/lib/python3.10/site-packages/django/core/management/__init__.py", line 386, in execute
2023-01-15T09:58:42.171999037Z  stdout      settings.INSTALLED_APPS
2023-01-15T09:58:42.172037801Z  stdout    File "/root/.local/lib/python3.10/site-packages/django/conf/__init__.py", line 92, in __getattr__
2023-01-15T09:58:42.172258152Z  stdout      self._setup(name)
2023-01-15T09:58:42.172714371Z  stdout    File "/root/.local/lib/python3.10/site-packages/django/conf/__init__.py", line 79, in _setup
2023-01-15T09:58:42.178001660Z  stdout      self._wrapped = Settings(settings_module)
2023-01-15T09:58:42.178137463Z  stdout    File "/root/.local/lib/python3.10/site-packages/django/conf/__init__.py", line 190, in __init__
2023-01-15T09:58:42.178182147Z  stdout      mod = importlib.import_module(self.SETTINGS_MODULE)
2023-01-15T09:58:42.178218028Z  stdout    File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
2023-01-15T09:58:42.178256316Z  stdout      return _bootstrap._gcd_import(name[level:], package, level)
2023-01-15T09:58:42.178291854Z  stdout    File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2023-01-15T09:58:42.178330796Z  stdout    File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2023-01-15T09:58:42.178365280Z  stdout    File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
2023-01-15T09:58:42.178398647Z  stdout    File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
2023-01-15T09:58:42.178505239Z  stdout    File "<frozen importlib._bootstrap_external>", line 883, in exec_module
2023-01-15T09:58:42.178542467Z  stdout    File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2023-01-15T09:58:42.178585312Z  stdout    File "/app/config/settings.py", line 20, in <module>
2023-01-15T09:58:42.178642304Z  stdout      from home.src.ta.config import AppConfig
2023-01-15T09:58:42.178738588Z  stdout    File "/app/home/__init__.py", line 3, in <module>
2023-01-15T09:58:42.178783739Z  stdout      from .tasks import app as celery_app
2023-01-15T09:58:42.178830783Z  stdout    File "/app/home/tasks.py", line 317, in <module>
2023-01-15T09:58:42.178880608Z  stdout      app.conf.beat_schedule = ScheduleBuilder().build_schedule()
2023-01-15T09:58:42.178913604Z  stdout    File "/app/home/src/ta/config.py", line 277, in build_schedule
2023-01-15T09:58:42.179005701Z  stdout      minute=item_conf["minute"],
2023-01-15T09:58:42.179046081Z  stdout  TypeError: string indices must be integers
```

**Comment by bbilly1** [comment]
Ayy, I see where that went wrong, very unfortunate. Never do last minute *improvements*... 

Quick fix would be to first pull v0.3.0, once that started successfully, you should be able to directly update to v0.3.1 without problems.

**Comment by bbilly1** [comment]
Thanks for reporting, this is now fixed in v0.3.2. You should be able to install that as a new install directly.
